core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1341,6 +1341,48 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-vsphere
     namespace: ci
+- from:
+    .awscred:
+      field: .awscred
+      item: jenkins-ci-iam
+    insights-live.yaml:
+      field: insights-live.yaml
+      item: insights-ci-account
+    pull-secret:
+      dockerconfigJSON:
+      - auth_field: token_image-puller_app.ci_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_field: auth
+        email_field: email
+        item: cloud.openshift.com-pull-secret
+        registry_url: cloud.openshift.com
+      - auth_field: auth
+        email_field: email
+        item: quay.io-pull-secret
+        registry_url: quay.io
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+    ssh-privatekey:
+      field: ssh-privatekey
+      item: jenkins-ci-iam
+    ssh-publickey:
+      field: ssh-publickey
+      item: jenkins-ci-iam
+    vmc.secret.auto.tfvars:
+      field: vmc.secret.auto.tfvars
+      item: vsphere-credentials
+  to:
+  - cluster_groups:
+    - non_app_ci
+    name: cluster-secrets-vsphere-discon
+    namespace: ci
```

Comment:
So this whole secret has 100% identical content to `cluster-secrets-vsphere`. Is that expected?

Reply:
yes, the same content with cluster-secrets-vsphere, the purpose to add new cluster profile(vspehre-discon) is only to use different lease resource. I'm not sure if I can reuse the secrets cluster-secrets-vsphere when configuring job where cluster_profile is vsphere-discon, or must define the section here?